DashAI/back/models/scikit_learn/numerical_wrapper_for_text.py

```diff
@@ -0,0 +1,102 @@
+import numpy as np
+from sklearn.feature_extraction.text import CountVectorizer
+
+from DashAI.back.core.schema_fields import BaseSchema, component_field, int_field
+from DashAI.back.dataloaders.classes.dashai_dataset import DashAIDataset
+from DashAI.back.models.scikit_learn.sklearn_like_model import SklearnLikeModel
+from DashAI.back.models.text_classification_model import TextClassificationModel
+
+
+class NumericalWrapperForTextSchema(BaseSchema):
+    """
+    NumericalWrapperForText is a metamodel that allows text classification using
+    tabular classifiers and a tokenizer.
+    """
+
+    tabular_classifier: component_field(
+        description="Tabular model used as the underlying model"
+        "to generate the text classifier.",
+        parent="TabularClassificationModel",
+    )
+    ngram_min_n: int_field(
+        description="Minimum n_gram to use in the vectorizer.",
+        default=1,
+        ge=1,
+    )
+    ngram_max_n: int_field(
+        description="Maximum n_gram to use in the vectorizer.",
+        default=1,
+        le=1,
```

Comment:
We will have to improve these descriptions in the future. Moreover, maybe we should have a mini-guide on how to write them.